bulletroboy/cage_simulation.py

```diff
@@ -38,6 +43,7 @@ def main():
     human_model = p.loadURDF(args.model_path, [0, 0, 1], useFixedBase=1)
 
     # EXOFORCE SETUP
+    operator = Operator(human_model)
```

Comment:
operator is already defined in the exoforce class, if you need to use it in the loop, call `exoforce.operator`

Reply:
done. :)